Automattic/jetpack
# [Plugin] Backup: Update menu item name and icon

#### Changes proposed in this Pull Request:
* Updated the menu title and icon
* These changes are temporary to provide clarity until the stand alone plugins are grouped under the Jetpack header

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Install the plugin and ensure the sidebar matches the 'after' image below

Before: 
<img width="162" alt="Screen Shot 2021-08-11 at 9 34 57 PM" src="https://user-images.githubusercontent.com/1992658/129124729-91f95807-d10c-4481-bd77-27a6f9c35344.png">

After:
<img width="157" alt="Screen Shot 2021-08-11 at 4 19 26 PM" src="https://user-images.githubusercontent.com/1992658/129098411-2f1379d8-8a90-4957-909f-f495864cf764.png">
